docs/recipes/make-stream-from-buffer.md

```diff
@@ -79,13 +79,14 @@ gulp.task('write-versions', function() {
   availableVersions.forEach(function(v) {
     // make a new stream with fake file name
     var stream = source('final.' + v);
+    
+    var streamEnd = stream;
```

Comment:
The former is possible but I wanted to make streamEnd more apparent.
I think the later is not possible because `stream.end()` is called async so we have to keep that reference.